certify-service/src/main/java/io/mosip/certify/services/SVGRenderUtils.java

```diff
@@ -0,0 +1,31 @@
+package io.mosip.certify.services;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+import io.ipfs.multibase.Multibase;
+
+public class SVGRenderUtils {
+    /**
+     * Generate SVG digest for @param svg image as per spec.
+     * ref: https://w3c-ccg.github.io/vc-render-method/#svgrenderingtemplate
+     *
+     * @param svg
+     * @return
+     */
+    public static String getDigestMultibase(String svg) {
```

Comment:
I see there is no logic specific to SVG rendering in this method, can we move this method to other general util class file ?

Reply:
@Piyush7034 Move this method to CredentialUtil class